CodeReview/Amie_code.py

```diff
@@ -9,9 +9,12 @@
 %matplotlib inline
 #Load data
 data = pd.read_csv('../Data/2014_PortTownsend.csv', parse_dates=['Date Time'])
+
 import plotly.plotly as py
 import plotly.graph_objs as go
+#Use date time index
 data_datetime = pd.DatetimeIndex(data['Date Time'])
+#Edit data adding new columns
 def f(data):
```

Comment:
Agree with Zak, also detail what data is and put it in """ """ instead of #, so that users can see it when they do f.?